vscode-lean4/src/config.ts

```diff
@@ -227,6 +227,10 @@ export function shouldShowInvalidProjectWarnings(): boolean {
     return workspace.getConfiguration('lean4').get('showInvalidProjectWarnings', true)
 }
 
+export function getOccurrenceHighlightStringFallback(): boolean {
+    return workspace.getConfiguration('lean4.editor').get('occurrenceHighlightWithStringFallback', true)
```

Comment:
The default for this should be `false`, not `true`.

Reply:
Indeed!